database/award_query.py

```diff
@@ -60,3 +60,37 @@ def _query_async(self):
             Award.team_list == ndb.Key(Team, team_key),
             Award.event == ndb.Key(Event, event_key)).fetch_async()
         raise ndb.Return(awards)
+
+
+class TeamYearTypeAwardsQuery(DatabaseQuery):
```

Comment:
If you add db queries, you're going to have to modify `CacheClearer` to make sure they get cleared at the right time.

Reply:
Why is this query even here? Doesn't seem like it's used.

Reply:
Ooh yea, I'll delete that one.